java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java

```diff
@@ -2,6 +2,8 @@
 
 package com.google.i18n.phonenumbers.nano;
 
+// @@protoc_insertion_point(imports_scope)
```

Comment:
Also I can't leave a comment on the jar files - we need to regenerate them before merging (to get the latest from the master merges), and squash the commits.